python/lsst/ts/MTAlignment/runAlignment.py

```diff
@@ -4,7 +4,7 @@
 def testComponent():
```

Comment:
This is named like a script but looks like a test. If this is supposed to be a "test script" I would recommend moving it to the `bin/` folder. For the IP, I would recommend turning it into a parameter for the script. This way you can easily run it on different environment and won't have to update it inline all the time. Port should probably be a parameter as well.